src/libEth/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(Eth Eth.cpp)
```

Comment:
Add a new library for functions relating to Eth stuff. I think we should put most stuff here basically